coralproject/elkhorn
# Add standard linting

## What does this PR do?

This sanitizes the code somewhat and exposes a few bugs since the editor is letting you know about anomalies. [JS Standard](https://github.com/feross/standard)
## How to test
- `npm install standard -g`
- follow the [instructions](https://github.com/feross/standard) for installing the plugin for your editor. atom is `apm install linter-js-standard`
- `npm install` (this will install standard into the project which seems to be required)
- your editor should now show warnings on the code when things are amiss
## Notable
- no more semicolons anywhere!
- spaces after the names of es6 class methods `render() {` becomes `render () {`
